### PR TITLE
release notes March 27 - April 3

### PR DESCRIPTION
**Pull request by qpetraroia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+* Starting in Kubernetes 1.23 AKS Metrics server deployment will start having 2 pods instead of 1 for HA, which will increase the memory requests of the system by 54Mb.
+* Kubernetes version 1.20 will be deprecated and removed from AKS on April 7th 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Node Pool Snapshot CLI experience is changing by April 6,  2022. The current nodepool snapshot commands i.e az `aks snapshot` will now be `az aks nodepool snapshot`.
+
+### Release notes
+
+* Features
+  * Windows Server 2022 is now available to use as your AKS cluster windows node when you enable `aks-preview` and register feature `Microsoft.ContainerService/AKSWindows2022Preview` and set `--os-sku Windows2022`.
```
Comment (PixelRobots) [inline CHANGELOG.md:20]: Would be good for a blog post here and also to include the vhd-notes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+* Kubernetes version 1.20 will be deprecated and removed from AKS on April 7th 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Node Pool Snapshot CLI experience is changing by April 6,  2022. The current nodepool snapshot commands i.e az `aks snapshot` will now be `az aks nodepool snapshot`.
+
+### Release notes
+
+* Behavioral changes
+  * The AKS run command now returns an error message with useful feedback if you cannot start a pod within 30 seconds.
+* Component Updates
+  * Azure Keyvault Secrets Provider has been updated to v1.1.0.
```
Comment (palma21) [inline CHANGELOG.md:22]: Link release notes pls

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+  * Azure Disk CSI driver has been updated to 1.14.
```
Comment (palma21) [inline CHANGELOG.md:23]: Link rel notes pls

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+  * Azure File CSI driver has been updated to 1.13.
```
Comment (palma21) [inline CHANGELOG.md:24]: Link rel notes please

**Review by palma21 — CHANGES_REQUESTED** [review]

**Review by miwithro — COMMENTED** [review]
@qpetraroia we pull the first announcement around the Upgrade for the security patches.

**Review by miwithro — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+* Starting in Kubernetes 1.23 AKS Metrics server deployment will start having 2 pods instead of 1 for HA, which will increase the memory requests of the system by 54Mb.
+* Kubernetes version 1.20 will be deprecated and removed from AKS on April 7th 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Node Pool Snapshot CLI experience is changing by April 6,  2022. The current nodepool snapshot commands i.e az `aks snapshot` will now be `az aks nodepool snapshot`.
```
Comment (palma21) [inline CHANGELOG.md:14]: This is now in the past.

**Review by palma21 — APPROVED** [review]

**Pull request merged**